noetic/distribution.yaml

```diff
@@ -2800,6 +2800,21 @@ repositories:
       url: https://github.com/anybotics/grid_map-release.git
       version: 1.6.4-1
     status: maintained
+  grpc:
```

Comment:
@BillWSY hmm, both rosdistro entry and package name are likely to clash with (or confuse people using) a future `rosdep` key for actual `grpc` binaries.

May I suggest `catkin_grpc` or `grpc_vendor` or anything along those lines for entry and package name? It also looks a lot like the source repository has some redundant nesting (i.e. `grpc` content could be at the root of the repo).

Reply:
Thanks for the suggestion. I prefer this name to keep it consistent with older ROS releases (we have released in indigo, kinetic, lunar, melodic). WDYT?